Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm

```diff
@@ -1385,7 +1387,8 @@ static void webProcessPoolHighDynamicRangeDidChangeCallback(CFNotificationCenter
         return;
 
     m_suppressEDR = suppressEDR;
-    screenPropertiesChanged();
+    constexpr bool skipStyleRecalc = true;
+    screenPropertiesChanged(skipStyleRecalc);
```

Comment:
If `suppressEDR` bit is not part of the ScreenProperties then screenPropertiesChanged message would only be needed if the computed `currentEDRHeadroom` actually changes here. But at least in my system in MiniBrowser `currentEDRHeadroom` always computes to "1" so we shouldn't be sending any messages here.

Reply:
Is your display set to XDR color profile? That might explain the difference

Reply:
That's not really the point. If nothing changes then we should not be sending ScreenProperties change messages.